suite/tests/CMakeLists.txt

```diff
@@ -547,6 +548,19 @@ function(add_exe test source)
     # it hard to use other DR builds w/ these apps
     SKIP_BUILD_RPATH ON)
   if (UNIX)
+    # for common.nativeexec takeover tests, we do not support -pie/pic. newer (unix-) distributions
```

Comment:
style violations here and below: multi-sentence comment must use capitilization and end punctuation (also preferred for single-sentence)

Reply:
Fixed